server/admin_management/api/app/routers/router_access_control_privilege.py

```diff
@@ -76,35 +76,32 @@ def create_access_control_privilege_many(
     )
 
     try:
-        audit_event_log.requesting_user = user_service.get_user_by_cognito_user_id(
-            requester.cognito_user_id
-        )
+        audit_event_log.requesting_user = requester
```

Comment:
Hi Catherine, I am wondering if this direct assign work (maybe work at runtime) as it seems they are different types.

Reply:
yeah, for "requester" I updated the type to be the Requester type from schema in audit config

Reply:
but I'll double check the type of "target_user"